backend/core/detection.py

```diff
@@ -122,6 +127,29 @@ def __init__(self, connection: Connection) -> NoReturn:
             self.redis = redis.Redis(host=REDIS_HOST, port=REDIS_PORT)
             ping_redis(self.redis)
 
+            self.rtrmanager = None
+            if RPKI_VALIDATOR_ENABLED == "true":
```

Comment:
if it's true false maybe they can be boolean already

Reply:
no they are strings as you can see in .env. It does not support booleans.